ospf6d/ospf6_route.c

```diff
@@ -1831,6 +1834,31 @@ void ospf6_brouter_show(struct vty *vty, struct ospf6_route *route)
 		OSPF6_PATH_TYPE_NAME(route->path.type), area);
 }
 
+DEFPY (debug_ospf6_route_all,
+       debug_ospf6_route_all_cmd,
+       "[no$no] debug ospf6 route all",
+       NO_STR
+       DEBUG_STR
+       OSPF6_STR
+       "Debug routes\n"
+       "Debug for all types of route calculation\n")
+{
+	if (!no) {
+		OSPF6_DEBUG_ROUTE_ON(OSPF6_DEBUG_ROUTE_TABLE);
+		OSPF6_DEBUG_ROUTE_ON(OSPF6_DEBUG_ROUTE_INTRA);
+		OSPF6_DEBUG_ROUTE_ON(OSPF6_DEBUG_ROUTE_INTER);
+		OSPF6_DEBUG_ROUTE_ON(OSPF6_DEBUG_ROUTE_MEMORY);
```

Comment:
for the lsa debug change, you added a "_ALL" macro, combining all of the existing flags values. why not do that for the DEBUG_ROUTE flags too, instead of hard-coding each individual test?